File: Clojure/abc199_a/main.clj
(defn solve [a b c]
  (cond (< (+ (* a a) (* b b)) (* c c)) "Yes"
        :else "No"))

(let [[a b c] (map #(Integer/parseInt %)
                   (clojure.string/split (read-line) #" "))]
  (println (solve a b c)))

File: Haskell/abc199_a/Main.hs
solve :: Int -> Int -> Int -> String
solve a b c =
  if a * a + b * b < c * c
    then "Yes"
    else "No"

main :: IO ()
main = do
  line <- getLine
  let [a, b, c] = map read $ words line :: [Int]
  putStrLn $ solve a b c
